HOMEWORKS/Chapter_05/Saeid/assets/js/script.js

```diff
@@ -0,0 +1,234 @@
+  photo.id = "photo_section"; // Create attribute id
+
+  let cardOne = document.createElement("div"); // card 01
+  cardOne.className = "card"; // Create attribute class
+  let photoOne = document.createElement("img"); // Create image
+  photoOne.src = "/HOMEWORKS/Chapter_05/assets/img/img_01.jpg"; // Image url
+  cardOne.appendChild(photoOne);
+
+  let cardTwo = document.createElement("div"); // card 02
+  cardTwo.className = "card"; // Create attribute class
+  let photoTwo = document.createElement("img"); // Create image
+  photoTwo.src = "/HOMEWORKS/Chapter_05/assets/img/img_02.jpg"; // Image url
+  cardTwo.appendChild(photoTwo);
+
+  let cardThree = document.createElement("div"); // card 03
+  cardThree.className = "card"; // Create attribute class
+  let photoThree = document.createElement("img"); // Create image
+  photoThree.src = "/HOMEWORKS/Chapter_05/assets/img/img_03.jpg"; // Image url
+  cardThree.appendChild(photoThree);
+
+  let cardFour = document.createElement("div"); // card 04
+  cardFour.className = "card"; // Create attribute class
+  let photoFour = document.createElement("img"); // Create image
+  photoFour.src = "/HOMEWORKS/Chapter_05/assets/img/img_04.jpg"; // Image url
+  cardFour.appendChild(photoFour);
+
+  let cardFive = document.createElement("div"); // card 05
+  cardFive.className = "card"; // Create attribute class
+  let photoFive = document.createElement("img"); // Create image
+  photoFive.src = "/HOMEWORKS/Chapter_05/assets/img/img_05.jpg"; // Image url
+  cardFive.appendChild(photoFive);
+
+  let cardSix = document.createElement("div"); // card 06
+  cardSix.className = "card"; // Create attribute class
+  let photoSix = document.createElement("img"); // Create image
+  photoSix.src = "/HOMEWORKS/Chapter_05/assets/img/img_06.jpg"; // Image url
+  cardSix.appendChild(photoSix);
+
+  photo.appendChild(cardOne); // card 01
+  photo.appendChild(cardTwo); // card 02
+  photo.appendChild(cardThree); // card 03
+  photo.appendChild(cardFour); // card 04
+  photo.appendChild(cardFive); // card 05
+  photo.appendChild(cardSix); // card 06
+  divOfPageWrapper.appendChild(photo); // Image section
+};
+photoSection();
+// End
+
+//Start download section
+const downloadSection = () => {
+  let download = document.createElement("section"); // create download section
+  download.id = "download_section"; // Create attribute id
+
+  let downTitle = document.createElement("p"); // create download title
+  TitleOfDown = document.createTextNode("Free Download at Start bootstrap!"); //title text
+  downTitle.appendChild(TitleOfDown);
+
+  let downButton = document.createElement("button"); // create header button
+  buttonOfDown = document.createTextNode("DOWNLOAD NOW!"); // button text
+  downButton.appendChild(buttonOfDown);
+
+  download.appendChild(downTitle); // title
+  download.appendChild(downButton); // button
+  divOfPageWrapper.appendChild(download); // download section
+};
+downloadSection();
```

Comment:
Since you are talking about downloading, i would love to see a PDF document that can be downloaded.
Search about that info and try to implement it in your code.